packages/twenty-front/src/modules/activities/components/ActivityRichTextEditor.tsx

```diff
@@ -306,6 +310,10 @@ export const ActivityRichTextEditor = ({
   useScopedHotkeys(
     '*',
     (keyboardEvent) => {
+      if (commandMenuPage !== CommandMenuPages.EditRichText) {
```

Comment:
It works but a more ideal solution would be to not mount that hook at all if we're editing, meaning we have the focus in the rich text editor. @bosiraphael